app-dashboard/js/view/layers/layer-statistic/osm-polygon-statistic.js

```diff
@@ -39,12 +39,25 @@ define([
                 polygonName = polygonName.replaceAll('T', '_');
                 polygonName = polygonName.split('.')[0];
                 polygonName = 'flood_' + polygonName;
+                let post_data = {
+                    'geometry': that.polygon,
+                    'name': polygonName,
+                    'reporting_date_time': new Date().toMysqlFormat(),
+                    'source': 'user',
+                    'station': $('#station').val()
+                };
+
+                if ($('#enable_forecast_date').is(':checked')) {
```

Comment:
could you move the form as backbone view? so it can return the postgres query when submitted